content/docs/1.13/performance-tuning.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Performance Tuning Guide
+description: Tweaking your Jaeger instance to achieve a better performance
+weight: 10
+---
+
+Jaeger was built from day 1 to be able to ingest huge amounts of data in a resilient way. To better utilize resources that might cause delays, such as storage or network communications, Jaeger buffers and batches data. When more spans are generated than Jaeger is able to safely process, spans might get dropped. However, the defaults might not fit all scenarios: for instance, agents running as a sidecar might have more memory constraints than agents running as a daemon in bare metal.
+
+## Adjusting the client settings
+
+### Settings `JAEGER_SAMPLER_TYPE` and `JAEGER_SAMPLER_PARAM`
+
+Together, the sampler type and parameter specify how often traces should be "sampled", ie, recorded and sent to the Jaeger backend. For applications generating a large number of spans, setting the sampling type to `probabilistic` and the value to `0.001` (the default) will cause traces to be reported with a 1/1000th chance. Note that the sampling decision is made at the root span and propagated down to all child spans.
+
+For applications with low to medium traffic, setting the sampling type to `const` and value to `1` will cause all spans to be reported. Similarly, tracing can be disabled by setting the value to `0`, while context propagation will continue to work.
+
+{{< info >}}
+Some clients support the setting `JAEGER_DISABLED` to completely disable the Jaeger Tracer. This is recommended only if the tracer is behaving in a way that causes problems to the instrumented application, as it will not propagate the context to the downstream services.
+{{< /info >}}
+
+### Setting `JAEGER_REPORTER_MAX_QUEUE_SIZE`
+
+Most of the Jaeger clients, such as the Java, Go, and C#, buffer spans in memory before batching them to the Jaeger Agent/Collector. The maximum size of this buffer is defined by the environment variable `JAEGER_REPORTER_MAX_QUEUE_SIZE` (default value: about `100` spans): the larger the size, the higher the potential memory consumption. When the instrumented application is generating a large number of spans, it’s possible that the queue will be full causing the client to discard the new spans (metric `jaeger_tracer_reporter_spans_total{result="dropped",}`).
+
+In the common scenarios, the queue (metric: `jaeger_tracer_reporter_queue_length`) will be close to empty, as spans are flushed at regular intervals to the agent or collector or when the concrete senders, such as the `UdpSender`, deem that a flush is needed.
+
+{{< info >}}
```

Comment:
I suggest not using the `info` highlight here, it's not _that_ important to this doc. Can simpler add it to the last paragraph.